.github/workflows/test_downstream.yml

```diff
@@ -117,8 +117,10 @@ jobs:
     name: Integration test on ${{ inputs.os }}
     runs-on: ${{ inputs.os }}
     steps:
-      - name: checkout repo
+      - name: Checkout repo
         uses: actions/checkout@v4
+      - name: Setup pixi
+        uses: prefix-dev/setup-pixi@v0.8.1
```

Comment:
could you please explain why we need to use setup-pixi action here and why it's better than old approach?

I fear that using setup-pixi we allow us to make accidently errors in moving binary and stuff, and still have a working pixi provided by setup-pixi which is not possible using the old approach

Reply:
I agree with an env variable for testing purposes